gaphor/__main__.py

```diff
@@ -0,0 +1,9 @@
+from gaphor import main
+
+
+def run_main():
+    main()
+
+
+if __name__ == "__main__":
+    run_main()
```

Comment:
No need for the run_main function. We can just call main directly.

Reply:
@amolenaar Good point, I'll clean that up! 👍